galite-core/src/main/kotlin/org/kopi/galite/visual/ui/vaadin/pivotTable/DPivotTable.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2013-2022 kopiLeft Services SARL, Tunis TN
+ * Copyright (c) 1990-2022 kopiRight Managed Solutions GmbH, Wien AT
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.kopi.galite.visual.ui.vaadin.pivotTable
+
+import com.vaadin.flow.component.dependency.CssImport
+import org.kopi.galite.visual.dsl.pivotTable.Position
+import org.kopi.galite.visual.pivotTable.MPivotTable
+import org.kopi.galite.visual.pivotTable.UPivotTable
+import org.kopi.galite.visual.pivotTable.VPivotTable
+import org.kopi.galite.visual.ui.vaadin.visual.DWindow
+
+import org.vaadin.addons.componentfactory.PivotTable
+
+@CssImport("./styles/galite/pivottable.css")
+class DPivotTable(private val pivotTable: VPivotTable) : DWindow(pivotTable), UPivotTable {
+
+  //---------------------------------------------------
+  // DATA MEMBERS
+  //---------------------------------------------------
+  private val model: MPivotTable = pivotTable.model // pivot table model
+  private val pivotData = PivotTable.PivotData()
+  private val pivotOptions = PivotTable.PivotOptions()
+  private val listeDimensionRows = mutableListOf<String>()
+  private val listeDimensionColumns = mutableListOf<String>()
```

Comment:
Rename `listeDimensionRows` to `rows`
Rename `listeDimensionColumns` to `columns`